rpc/src/rpc.rs

```diff
@@ -469,19 +466,18 @@ impl JsonRpcRequestProcessor {
             );
             ClusterInfo::new(contact_info, keypair, socket_addr_space)
         });
-        let tpu_address = cluster_info
-            .my_contact_info()
-            .tpu(connection_cache.protocol())
-            .unwrap();
+        let my_tpu_address = cluster_info.my_contact_info().tpu(Protocol::QUIC).unwrap();
```

Comment:
What's the reason that this is OK to hardcode to `QUIC`?

Reply:
My understanding is that UDP is deprecated to TPU traffic and `DEFAULT_CONNECTION_CACHE_USE_QUIC == true`. So before these changes `ConnectionCache` was always created for QUIC and all the tests were running for this protocol. The new client, tpu-client-next, doesn't have UDP option since it is deprecated so I hard-coded here QUIC. I've added comment here.